skbio/io/fasta.py

```diff
@@ -557,6 +563,10 @@
 
 @register_sniffer('fasta')
 def _fasta_sniffer(fh):
+
+    if _count_blank_lines(fh) > 5:
```

Comment:
Ah you're right. I refactored it when I pushed it up to `_base.py` without thinking about it. Technically we have duplicated logic here between `fasta.py` and `fastq.py` because of the `> 5`. What do you think about calling the funtion `_has_too_many_blanks` and moving the `> 5` inside?